pkg/proto/types.go

```diff
@@ -2087,6 +2087,10 @@ func (p *ProofsV1) Valid() error {
 	return nil
 }
 
+func (p *ProofsV1) MultiSigned() bool {
```

Comment:
What if there is an empty proofs list? Such transaction is also must be validated by verifier. 
I think better name for this function is `SimpleSignature` with the opposite meaning. And it have to check not just the length of proofs, but the position of signature (it must be 0) and the length of signature itself.